NomadAudio/src/Win32/WindowsDriverRegistry.cpp

```
@@ -0,0 +1,54 @@
+#include <iostream>
+
+namespace Nomad {
+namespace Audio {
+
+void RegisterPlatformDrivers(AudioDeviceManager& manager) {
+    std::cout << "[AudioDriverRegistry] Registering Windows drivers..." << std::endl;
+    
+    // Register WASAPI Exclusive
+    try {
+        auto exclusive = std::make_unique<WASAPIExclusiveDriver>();
+        if (exclusive->initialize()) {
```

Clang (14.0.6)

[warning] if with identical then and else branches

(bugprone-branch-clone)